### PR TITLE
README relocation

### PR DESCRIPTION
**Pull request by JacobMGEvans**
The CI/CD would try to checkout to another branch for creating the Changesets PR or adding to it. When it attempted to checkout after the README was copied from the root of the monorepo to the Wrangler package it would have an unstaged file, then abort the checkout and fallback to creating a new branch to handle the new file.
Solution:
- Moved Wrangler README to Wrangler package
- Created Monorepo README in root of repo
- Removed the script for copying README from root into Wrangler package

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,51 +1,11 @@
+# Wrangler Monorepo Documentation
+
 ## 🤠 wrangler
 
-`wrangler` is a command line tool for building [Cloudflare Workers](https://workers.cloudflare.com/).
+`wrangler` is a command line tool for building [Wrangler Documentation](./packages/wrangler/README.md).
```
Comment (threepointone) [inline README.md:5]: ?? wrangler isn't a tool for making documentation. huh?

Comment (threepointone) [reply README.md:5]: This doc needs to be fleshed out before it lands, it's very sparse right now and doesn't say anything.

**Review by threepointone — CHANGES_REQUESTED** [review]
Needs a better README at the root.

File: README.md (hunk 1 above)
Comment (JacobMGEvans) [reply README.md:5]: True, wasn't sure what to put for the monorepo documentation.

Comment (JacobMGEvans) [reply README.md]: > ?? wrangler isn't a tool for making documentation. huh?

This was an error. Updated it.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -39,9 +39,7 @@ jobs:
         id: changesets
         uses: changesets/action@v1
         with:
-          publish:
-            - cp README.md packages/wrangler/README.md
```
Comment (petebacondarwin) [inline .github/workflows/release.yml:43]: There is a copy command in the prerelease too: https://github.com/cloudflare/wrangler2/blob/485e0dcf9966b03c641cdbf38c6c626e6469d1c1/.github/workflows/prereleases.yml#L35-L36

**Comment by petebacondarwin** [comment]
It is worth thinking about the readers of these READMEs.

The wrangler one will be seen by users installing the tool (i.e. via npm or browsing node_modules).
The monorepo one is more likely read by developers looking to contribute to the project (IMO).

**Review by petebacondarwin — APPROVED** [review]

**Pull request merged**